documentation/src/docs/recovery-procedures/lts-auto-recovery-steps.md

```diff
@@ -0,0 +1,56 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
```

Comment:
Backup
1. Stop traffic
2. Enable admin port
3. Scale down controller
4. call flush to storage
5. Enable retain on PV so that they won't be deleted
6. Save PV/PVC files
7.  uninstall Pravega etc.

Restore
1. Reattach PV and PVC. 
2. start Pravega
3. start traffic